watcher/watchertest/notify.go

```diff
@@ -6,26 +6,28 @@ package watchertest
 import (
 	"time"
 
+	"github.com/juju/loggo"
 	jc "github.com/juju/testing/checkers"
 	gc "gopkg.in/check.v1"
-	tomb "gopkg.in/tomb.v1"
+	tomb "gopkg.in/tomb.v2"
 
 	"github.com/juju/juju/testing"
 	"github.com/juju/juju/watcher"
 )
 
+var logger = loggo.GetLogger("veebers")
```

Comment:
delete me?

Reply:
yep, damn I missed a couple of things in the debugging cleanup.